code/modules/client/preference_setup/loadout/loadout_xeno/unathi.dm

```diff
@@ -71,6 +71,14 @@
 	sort_category = "Xenowear - Unathi"
 	flags = GEAR_HAS_NAME_SELECTION | GEAR_HAS_DESC_SELECTION | GEAR_HAS_COLOR_SELECTION
 
+/datum/gear/accessory/sinta_surcoat
+	display_name = "ghazak"
```

Comment:
The design is quite literally the flag, and it looks somewhat tacky. I'm opposed to this being added in its current iteration. A better implementation of something like a shoulder pauldron or cape would work well for adding the colors without it being the centerpiece.